acceptance/auth/bundle_and_profile/databricks.yml

```diff
@@ -0,0 +1,14 @@
+bundle:
+  name: test-auth
+
+workspace:
+  host: $DATABRICKS_HOST
+
+targets:
+  dev:
+    default: true
+    workspace:
```

Comment:
nit: We don't have to override the same host here since it's the default target

Reply:
Indeed, but I just made it explicit